apps/service-portal/src/store/modules.ts

```diff
@@ -75,7 +75,7 @@ export const modules: Record<ModuleKeys, ServicePortalModule> = {
   assets: assetsModule,
   applications: applicationsModule,
   accessControl: accessControlModule,
-  licenses: licensesModule,
+  licenses: licenseModule,
```

Comment:
Should this be license instead of licenseS ?

Reply:
It's a module for multiple licenses so it should probably be plural. 👀